domain/app/models/exercise.rb

```diff
@@ -183,6 +183,13 @@ def limited?
     navigable_parent.limited_for?(self)
   end
 
+  def files_for(current_content)
```

Comment:
Is there any test for this code? I think you have just refactored an untested method.

Reply:
I see. I'd **add** a few simple tests in domain for the `files_for` in that case. 

Please proceed, thanks.